rotkehlchen/data/nodes.json

```diff
@@ -54,5 +54,29 @@
         "owned": false,
         "active": true,
         "blockchain": "ETH"
+    },
+    {
```

Comment:
It would probably make sense to modify this file and change it to be a mapping by chain id and move the `blockchain` part out as the key